kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyFrontendHandler.java

```diff
@@ -489,6 +489,8 @@ void inConnecting(
             pipeline.addFirst("ssl", handler);
         });
 
+        LOGGER.debug("Configured broker channel pipeline: {}", pipeline);
```

Comment:
I realised that we don't log the pipeline. 

Note to self: `isDebugEnabled()`

Reply:
Actually no need for `isDebugEnabled()`, since `toString` will only be called on the `pipeline` when debug is enabled.